doc/CHANGES.md

```diff
@@ -65,6 +65,8 @@
   are no requests (Liz Conlan)
 * Switch to Trusty as the preferred OS for Travis CI and use Debian Wheezy as
   the new Vagrant default (Liz Conlan)
+* Fix bug which prevented stats from being displayed in the admin interface
```

Comment:
I'd say we don't need a changelog entry as there wasn't a release introducing the bug